README.md

```diff
@@ -349,6 +349,16 @@ all of your arguments must be keyword arguments.
 Note: This is a limitation of the current `significant_kwargs` implementation,
 and totally fixable if someone wants to submit a pull request.
 
+### async_or_eager
```

Comment:
:+1: 

Could we list these two below the `delay_or` versions? My impression is that most folks will continue to use the simpler signature of `delay_or`.